pkg/config/config.go

```diff
@@ -1479,3 +1538,24 @@ func (c *NetworkConfig) CNIManagerShutdown() {
 func (c *Config) SetSingleConfigPath(singleConfigPath string) {
 	c.singleConfigPath = singleConfigPath
 }
+
+// Equals checks whether to runtimes tables are equal.
+func (r Runtimes) Equals(o Runtimes) bool {
```

Comment:
we could use RuntimesEqual in pkg/config/template.go instead

Reply:
Thanks. I have switched to this. One thing that I was concerned about was to ignore ordering in the `RuntimeHandler` members that are slices but this may be over engineering things at this stage.

Note that I call `RuntimesEqual` in `reload.go` directly instead of wrapping it here.

Reply:
worst case, we get a false positive because the members are in a different order, and we reload the config. we'll spend some unnecessary cpu time, but it doesn't seem like a horrible case. I think the code succinct-ness is worth it